### ISSUE TITLE
TripChangeRequest - parameters missing

### ISSUE THREAD

**Issue by trurlurl** [VDVde/OJP#383]
I still think that the TripChangeRequest - #381 - needs to know many of the inital parameters that were used to find the trip to be changed. 

Currently, there is only *TripContentFilterGroup* and *OperatorFilter*. 

The easiest would probably be to have the complete *TripParamStructure*, then you can decide which parameters to use and which not. I think the parameter element should be compulsory and the examples have to be adapted.

I might do the PR, but only if @herlitze agrees on the problem, solution, and procedure.

**Comment by herlitze** [comment]
@trurlurl I agree with adding the TripParamStructure, but I think it should be optional as it is optional in the TripRequest as well. I would appreciate it if you could take over the PR, but I can of course correct my carelessness myself.

**Comment by trurlurl** [comment]
Solved by the referring PR.
